README.md

```diff
@@ -49,6 +49,12 @@ the [Wikidata project](https://www.wikidata.org/).
 
 ## Release notes
 
+### 0.13.0 (2024-12-10)
```

Comment:
Is this intentionally set to tomorrow’s date? ^^

Reply:
yes - I haven't had so many same-day reviews :) I figure if the review gets through I can hit the 'create release' button tomorrow.

Reply:
LGTM, but IMHO we shouldn’t merge it today then.